hub/src/server/drivers/S3Driver.js

```diff
@@ -62,31 +71,57 @@ class S3Driver implements DriverModel {
     return `https://${this.bucket}.s3.amazonaws.com/`
   }
 
-  createIfNeeded () {
-    this.s3.headBucket( { Bucket: this.bucket }, (error) => {
-      if (error && error.code === 'NotFound') { // try to create
+  async createIfNeeded () {
+    try {
+      await this.s3.headBucket({ Bucket: this.bucket }).promise()
```

Comment:
should this use `this.s3BucketExists()`  instead?

Reply:
I'm also curious why `this.s3BucketExists()` isn't being used.

Reply:
Yep, fixed